.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "4"
-  - "node"
+  - '12'
```

Comment:
Please set the version to 10, which is the oldest actively supported Node.js version.  12 is too new.

Reply:
done